docs/spark/tutorials/get-started.md

```diff
@@ -1,9 +1,11 @@
 ---
 title: Get started with .NET for Apache Spark
 description: Discover how to run a .NET for Apache Spark app using .NET Core on Windows, MacOS, and Ubuntu.
```

Comment:
```suggestion
description: Discover how to run a .NET for Apache Spark app using .NET Core on Windows, macOS, and Ubuntu.
```

Reply:
Also, is there a reason Ubuntu is called out directly? Should this just be Linux?

Reply:
The .NET for Spark worker package targets the Ubuntu distro.